zerver/tests/test_example.py

```diff
@@ -368,15 +368,14 @@ def test_capturing_queries(self) -> None:
         hamlet = self.example_user("hamlet")
         cordelia = self.example_user("cordelia")
 
-        with queries_captured() as queries:
+        with self.assert_query_count(15):
```

Comment:
I think we should **update the comment on line 360**. We no longer directly use the `queries_captured()` context manager. So, it should be updated accordingly to avoid confusion.